components/blitz/resources/ome/services/blitz-servantDefinitions.xml

```diff
@@ -390,6 +390,11 @@
       <constructor-arg value="${omero.graphs.wrap}"/>
   </bean>
 
+  <bean class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+    <property name="staticMethod" value="ome.services.graphs.GroupPredicate.setSecuritySystem"/>
+    <property name="arguments"><list><ref bean="securitySystem"/></list></property>
```

Comment:
As a bug fix, I can live with this, but in general, static state will get us into trouble and I actively try to remove it where possible. More optimally (eventually) would be to inject either a GroupPredicate or worst case a GroupFactoryFactory where you need it.